Tests/UnitTests/OrderTests.swift

```diff
@@ -0,0 +1,89 @@
+//
+// OrderTests.swift
```

Comment:
Why is this a new file?

Reply:
Where is the old order tests file?

Reply:
We're missing tests that the order object serializes with the correct wire field names too. See the original [OrderTests.swift](https://github.com/button/button-merchant-ios/blob/master/Tests/UnitTests/Models/OrderTests.swift#L69)

Reply:
The required arguments are non-nil so Swift's type safety won't allow it.